ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -76,6 +79,8 @@ def _underlay_links(self) -> list:
                         "ipv6_enable": get(uplink, "ipv6_enable"),
                         "structured_config": get(uplink, "structured_config"),
                     }
+                    if default(get(uplink, "sflow"), self.shared_utils.fabric_sflow_downlinks) is not None:
+                        link["sflow"] = {"enable": default(get(uplink, "sflow"), self.shared_utils.fabric_sflow_downlinks)}
```

Comment:
Since the `strip_empties_from_dict` will remove a None value, we can just insert the downlinks value directly.
```suggestion
                        "ipv6_enable": get(uplink, "ipv6_enable"),
                        "sflow": self.shared_utils.fabric_sflow_downlinks,
                        "structured_config": get(uplink, "structured_config"),
                    }
```